VisualStudio.gitignore

```diff
@@ -151,7 +151,7 @@ publish/
 # Publish Web Output
 *.[Pp]ublish.xml
 *.azurePubxml
-# TODO: Comment the next line if you want to checkin your web deploy settings
```

Comment:
:thumbsup: to some other prefix here. I'm not really sure if there's a better way here to indicate that you _can_ introduce an optional rule here, but `Note:` is a fine substitute.